### ISSUE TITLE
REPL Evaluation

### ISSUE THREAD

**Issue by ghost** [JordanMartinez/purescript-jordans-reference#384]
## Context

```
Section:
  Getting Started

Issue Type:
  Bug

Folder/File(s):
  - Getting Started/The REPL.md
```

## Body

I guess

```
> :kind (Int -> Int)
Type -> Type
```

should be

```
> :kind (Int -> Int)
Type
```

**Comment by JordanMartinez** [comment]
Ah... Whoops! That's correct.

**Comment by JordanMartinez** [comment]
Fixed in 025412b

Unfortunately, the Spago upgrade is blocking the release.
